xml_inscriptions/transcribed/NJ.Princ.PU.AM.2003-129A-B.xml

```diff
@@ -86,9 +86,20 @@
                         </summary>
 
                         <origin>
+
+                            <date notBefore="0199" notAfter="0399">2nd to 4th century CE</date>
+                            <placeName ref="asia.asia_minor">Antioch, Turkey</placeName>
+
                             <date notBefore="0050" notAfter="0250">first to third century CE</date>
                             <placeName ref="xx"></placeName>
+
                         </origin>
+                        <acquisition>
```

Comment:
We don't really use `<acquisition>`, better to use `<provenance type="transferred" subtype="given">` vel sim., which is more in line with the EpiDoc guidelines: see http://www.stoa.org/epidoc/gl/latest/supp-histlocations.html.